File: 3. Baekjoon/1225. 이상한 곱셈.cpp
#include <iostream>
#include <numeric>
#include <string>

using namespace std;

int main() {
    string A, B;
    cin >> A >> B;

    long long sum_A = 0, sum_B = 0;

    for (char a : A) {
        sum_A += a - '0';
    }

    for (char b : B) {
        sum_B += b - '0';
    }

    long long result = sum_A * sum_B;

    cout << result << endl;

    return 0;
}

File: 3. Baekjoon/1225. 이상한 곱셈.py
import sys

input = sys.stdin.readline

if __name__ == "__main__":
    A, B = input().split()

    sum_A = sum(int(a) for a in A)
    sum_B = sum(int(b) for b in B)

    result = sum_A * sum_B

    print(result)
